lib/globals.css

```diff
@@ -193,12 +198,12 @@ table {
   word-break: break-word;
 }
 
-.callout-container > .callout-icon {
+.callout-container>.callout-icon {
```

Comment:
Are these intentionals? I though 1st one was the way to do it? Is this auto formatted by the prettier and is this the recommended style?